config/locales/cz.yml

```diff
@@ -80,106 +80,107 @@ cz:
-      first_time_link: Start here. →
-      heading: About CrimethInc.
+      first_time_text: Slyšíš o CrimethInc poprvé. ?
+      first_time_link: Začni tady. →
+      heading: O CrimethInc.
       description: |
-        CrimethInc. is a rebel alliance—a decentralized network pledged to anonymous collective action—a breakout from the prisons of our age. We strive to reinvent our lives and our world according to the principles of self-determination and mutual aid.
+        CrimethInc. Je odbojová allince a decentralizovaná síť která se zavázala k anonymní kolektivní akci a útěku z vězení našeho času. Snažíme se znovuobjevit náš život ve světe s ohledem na principy sebeurčení a vzájemné pomoci.
 
-        We believe that you should be free to dispose of your limitless potential on your own terms: that no government, market, or ideology should be able to dictate what your life can be. If you agree, _let’s do something about it._
-      more_link: More about CrimethInc. →
-      donate_button_text: Support this site
+        Věříme že by si měl mít možnost volně využít svůj nekonečný potenciál tak jak uznáš za vhodné. Žádná vláda, trh , nebo ideologie by ti neměla diktovat jaký bude tvůj život. Jestli souhlasíš, _Tak s tím pojďme něco udělat._
+      more_link: Víc o CrimethInc. →
+      donate_button_text: Podpoř nás.
 
     contact:
       newsletter:
-        heading: Email Updates
-        description: Sign up here to receive regular email updates. _(Processed by Mailchimp.)_
-        signup_form_placeholder: What’s your email address?
-        signup_form_select_placeholder: You want a lot of emails? Or a little?
-        signup_form_select_option_lot:    Every little thing. (Many emails)
-        signup_form_select_option_little: Just the big stuff! (Fewer emails)
+        heading: Aktualizace Emailem
+        description: Přihlaš se pro odběr pravidelných zpráv z našeho webu. _(Zpracováno přes MailChimp.)_
+        signup_form_placeholder: Jaký je tvůj email?
+        signup_form_select_placeholder: Chceš hodně emailů? Nebo málo?
+        signup_form_select_option_lot:    Každou maličkost! (Many emails)
+        signup_form_select_option_little:  Jen to důležité! (Fewer emails)
         signup_button_text: Sign Up
       social:
-        heading: Find Us Elsewhere
-        description: Keep up with our activities on the following additional platforms.
+        heading: Sleduj nás jinde
+        description:  Sleduj naše aktivity na následujících platformách.
 
     mantra:
-      - If your heart is free, the ground you stand on is liberated territory. Defend it.
-      - Only those prepared to go too far will learn how far they can go.
-      - Desperate measures are long overdue.
-      - Cradle the seed, even in the volcano’s mouth.
-      - The future is in chains within you — set it free.
-      - A spark for every powder keg.
+      - Jestli je tvé srdce volné, zem na které stojíš je již osvobozená. Nenech si ji vzít.
```

Comment:
I don't read Czech, so this is more just me being curious. Feel free to ignore this comment:

Is this an idiom in Czech, just wondering as the dictionary I am using (https://slovnik.seznam.cz/preklad/cesky_anglicky/Nenech) kind of translated this one odd

Reply:
it's not idiom, just freely translated from this "If your heart is free, the ground you stand on is liberated territory. **Defend it"** Only difference is that i changed the bold part from defend it which would be something like "Braň ho" and just sounded wrong in this context so i choose more freeflow tranlastion which didn't change the meaning , i don't thing i can translate it back without it sounding dumb but it would be something like "Don't let them take it from you"